FridgeExercise/Program.cs

```diff
@@ -1,2 +1,242 @@
-﻿// See https://aka.ms/new-console-template for more information
-Console.WriteLine("Hello, World!");
+﻿using FridgeExercise;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using static FridgeExercise.Item;
+using static System.Net.Mime.MediaTypeNames;
+
+namespace FridgeExercise
+{
+	
+	internal class Program
+	{
+		public void SortheRefrigerators(List<Refrigerator> AllRefrigerators)
```

Comment:
```suggestion
		public void SortRefrigeratorsByAvailableSpace(List<Refrigerator> AllRefrigerators)
```

It's important to be implicit in your naming, for the sake of readability :)